extensions/twitter/README.md

```diff
@@ -0,0 +1,36 @@
+# Twitter extension
+This document defines how to use `twitter` extension in AsyncAPI documents.
+
+## Twitter Object
```

Comment:
please have a look at "what's an extension"

Reply:
I didn't get it ,can you please elaborate a little.

Reply:
oh sorry, my comment had more content 🤔 

please look at https://github.com/asyncapi/extensions-catalog/pull/112#pullrequestreview-1182610285 as this document needs restructure

Reply:
thanks this will help a lot 😀

Reply:
I think only this one is missing

Reply:
@derberg just to clarify I need to add the elaborate what is an extension?